helpers/filterSeats.js

```diff
@@ -86,8 +111,56 @@ module.exports = function(dust) {
       });
     }
 
+    function reorderRates(rates, replyIndex) {
+      var bandTypes = {
+        'gold': [],
+        'silver': [],
+        'bronze': []
+      };
```

Comment:
@sam-cook there is no test coverage for this helper as far as I can see.
@owennicol this is an interesting point. I'll try to make this object dynamic.